tox.ini

```diff
@@ -1,42 +1,50 @@
 [tox]
-envlist = py36-flake8, py36-pylint, py36-test, py36-bandit
+envlist = py{36,37-dev,py3},flake8,pylint,test,bandit
 skipsdist=True
+skip_missing_interpreters = {env:TOX_SKIP_MISSING_INTERPRETERS:True}
+app = nim
 
 [testenv]
-ignore_errors = False
-basepython = python3.6
-
-[testenv:py36-flake8]
 use_develop = True
-commands = flake8 --max-line-length 99 --max-complexity 12 nim
+
+[testenv:flake8]
+commands = flake8 --max-line-length 99 --max-complexity 12 {[tox]app}
 deps = flake8
 
-[testenv:py36-pylint]
-use_develop = True
+[testenv:pylint]
 # Ignore return code of Pylint for now to not make the build fail. Remove the
 # dash once we've fixed all the issues.
-commands = - pylint nim
+commands = - pylint {[tox]app}
 deps = pylint
 
-[testenv:py36-test]
-use_develop = True
+[testenv:test]
 commands =
     pytest --cov=./ {posargs}
 deps =
     mock
     pytest
     pytest-cov
 
-[testenv:py36-bandit]
-use_develop = True
+[testenv:bandit]
 # Ignore return code of Bandit for now to not make the build fail. Remove the
 # dash once we've fixed all the issues.
-commands = - bandit -r nim
+commands = - bandit -r {[tox]app}
 deps = bandit
 
 [testenv:cleanup]
 commands =
+    rm -f .coverage
     rm -rf .tox/
     rm -rf .pytest_cache/
-    rm -rf nim.egg-info/
-whitelist_externals = rm
+    rm -rf {[tox]app}.egg-info/
+    find . -path '*/__pycache__/*' -delete
+    find . -type d -name '__pycache__' -delete
```

Comment:
2 commands because the last one will fail if the `__pycache__` directories aren't empty, so the first command will remove the files and the second one will remove the (now empty) directories.